scripts/functional-tests.sh

```diff
@@ -68,6 +68,7 @@ echo -e "\n\n\n" | ilab config init
 
 # Enable Debug in func tests
 sed -i.bak -e 's/log_level:.*/log_level: DEBUG/g;' config.yaml
+sed -i -e 's/debug_level:.*/debug_level: 2/g;' config.yaml
```

Comment:
```suggestion
sed -i.bak -e 's/debug_level:.*/debug_level: 2/g;' config.yaml
```

Let's re-use what other parts of the script do.

Reply:
Also, this will make `test_no_chat_logs` fail since it checks that `_base_client.py` should **not** be logged when running INFO level.

Reply:
FTR, I left out the second `.bak` to keep the original bak file.